File: PYTHON/Program/Produk.py
class Produk:
    #constructor
    def __init__(self, id_produk, merk, harga, stok):
        self.id_produk = id_produk
        self.merk = merk
        self.harga = harga
        self.stok = stok

    # getter dan setter
    def getid_produk(self):
        return self.id_produk

    def getMerk(self):
        return self.merk

    def setMerk(self, merk):
        self.merk = merk

    def getHarga(self):
        return self.harga

    def setHarga(self, harga):
        self.harga = harga

    def getStok(self):
        return self.stok

    def setStok(self, stok):
        self.stok = stok

File: CPP/Program/Produk.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

class Produk
{
private:
    int id_produk;
    string merk;
    int harga;
    int stok;

public:
    Produk()
    {
    }
    //constructor
    Produk(int id_produk, string merk, int harga, int stok)
    {
        this->id_produk = id_produk;
        this->merk = merk;
        this->harga = harga;
        this->stok = stok;
    }
    
    //getter dan setter
    int getid_produk()
    {
        return id_produk;
    }

    string getMerk()
    {
        return merk;
    }

    void setMerk(string merk)
    {
        this->merk = merk;
    }

    int getHarga()
    {
        return harga;
    }

    void setHarga(int harga)
    {
        this->harga = harga;
    }

    int getStok()
    {
        return stok;
    }

    void setStok(int stok)
    {
        this->stok = stok;
    }
};